.storybook/main.ts

```diff
@@ -1,21 +1,19 @@
-import type { StorybookConfig } from "@storybook/react-vite";
-
-const config: StorybookConfig = {
+const config = {
   stories: ["../src/**/*.mdx", "../src/**/*.stories.@(js|jsx|ts|tsx)"],
   addons: [
     "@storybook/addon-links",
     "@storybook/addon-essentials",
     "@storybook/addon-interactions",
-  ],
-  core: {
-    builder: {
-      name: "@storybook/builder-vite",
+    {
+      name: "@storybook/addon-postcss",
       options: {
-        viteConfigPath: "./.storybook/vite.config.ts",
```

Comment:
I would recommend using the vite builder as we are using it for our project
and vite builder requires its own config that i created under .storybook/vite.config.ts